core/src/main/java/org/apache/spark/api/java/function/package.scala

```diff
@@ -22,4 +22,4 @@ package org.apache.spark.api.java
  * these interfaces to pass functions to various Java API methods for Spark. Please visit Spark's
  * Java programming guide for more details.
  */
-package object function 
+package object function
```

Comment:
I'm pretty sure it's already there, because otherwise git would show a red mark at the end. Also I verified locally by running wc (if I remove the trailing new line, wc result goes from 25 to 24).

But more importantly, why are we wasting time here?